requirements/requirements_ansible.in

```diff
@@ -4,29 +4,37 @@ packaging
 azure-cli-core==2.0.35
 azure-cli-nspkg==3.0.2
 azure-common==1.1.11
-azure-mgmt-batch==4.1.0
-azure-mgmt-compute==2.1.0
-azure-mgmt-containerinstance==0.4.0
+azure-mgmt-authorization==0.51.1
```

Comment:
Might as well update the above comment to link to https://github.com/ansible/ansible/blob/devel/packaging/requirements/requirements-azure.txt, since it's really not using the 2.7 version. Or maybe link to the stable-2.8 branch, if the point is to comment with what Ansible version it came from

Reply:
oh right, I fixed that, but elided it from the commit due to pip-compile rewriting other comments. Will update.